Tests/AppTests/PackageController+routesTests.swift

```diff
@@ -697,18 +705,35 @@ class PackageController_routesTests: AppTestCase {
         Current.fetchDocumentation = { _, uri in
             // embed uri.path in the body as a simple way to test the requested url
             .init(status: .ok,
-                  headers: ["content-type": "application/octet-stream"],
+                  headers: ["content-type": "application/json"],
                   body: .init(string: uri.path))
         }
 
         // MUT
         try app.test(.GET, "/owner/package/1.2.3/theme-settings.json") {
             XCTAssertEqual($0.status, .ok)
-            XCTAssertEqual($0.content.contentType?.description, "application/octet-stream")
+            XCTAssertEqual($0.content.contentType?.description, "application/json")
             XCTAssertEqual($0.body.asString(), "/owner/package/1.2.3/theme-settings.json")
         }
     }
 
+    func test_linkableEntites() throws {
```

Comment:
This test should also go if we don't end up serving the `linkable-entities.json`.